presto-native-execution/presto_cpp/main/TaskManager.cpp

```diff
@@ -207,15 +207,16 @@ void checkSplitsForBatchTask(
 }
 
 struct ZombieTaskStats {
-  const std::string taskId;
-  const std::string taskInfo;
-
-  explicit ZombieTaskStats(const std::shared_ptr<exec::Task>& task)
-      : taskId(task->taskId()), taskInfo(task->toString()) {}
-
-  std::string toString() const {
-    return SystemConfig::instance()->logZombieTaskInfo() ? taskInfo : taskId;
-  }
+  const std::string info;
+  const long numExtraReferences;
+
+  ZombieTaskStats(
```

Comment:
Nit: Why was `explicit` removed? This should still be used for the construction during the emplace call like before?

Reply:
explicit is was used for single-argument constructors, so there would be no implicit type conversion.